development/osaka/eip-7805.md

```diff
@@ -0,0 +1,9 @@
+# EIP-7805 Metrics
+
+| Name | Metric type | Usage | Sample collection event | Labels | Buckets |
+|------|-------------|-------|-------------------------|--------|---------|
+|`execution_inclusion_list_transactions_received_in_payload_total`| Counter | Total number of inclusion list transactions received in payload | On receiving payload | | |
```

Comment:
This should be equal to `beacon_inclusion_list_transactions_sent_to_payload_total`, right?